### PR TITLE
Added quick-start.md in spanish and link into sidebar in config.ts

### PR DESCRIPTION
**Pull request by felipe300**
## Changes

- Add  quick-start.md in src/pages/es
- Add sidebar "Comienzo rápido" in src/config.ts
- Fix a typo in installation.md in the folder src/pages/es

## Testing

- I check the files in my local environment with no problems.

## Docs

- docs/src/pages/es/installation.md
- docs/src/pages/es/quick-start.md
- docs/src/config.ts.md

### CODE REVIEW

File: docs/src/pages/es/installation.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ Arriba hay un ejemplo de la sintaxis del componente de Astro, que comprende HTML
 
 Puedes crear más páginas en el directorio `src/pages` y Astro utilizará los archivos creados para generar nuevas páginas de tu sitio web. Por ejemplo, Al crear el archivo `about.astro` en `src/pages/about.astro` (reusando el fragmento de código anterior), Astro generará una página HTML en la dirección URL: `http://localhost/about`.
 
+<<<<<<< HEAD
+=======
+
+>>>>>>> c591bd07 (Added installation.md in spanish)
```
Comment (matthewp) [inline docs/src/pages/es/installation.md:140]: Merge conflict

File: docs/src/pages/es/installation.md [hunk 2]
```diff
@@ -134,6 +134,10 @@ Arriba hay un ejemplo de la sintaxis del componente de Astro, que comprende HTML
 
 Puedes crear más páginas en el directorio `src/pages` y Astro utilizará los archivos creados para generar nuevas páginas de tu sitio web. Por ejemplo, Al crear el archivo `about.astro` en `src/pages/about.astro` (reusando el fragmento de código anterior), Astro generará una página HTML en la dirección URL: `http://localhost/about`.
 
+<<<<<<< HEAD
```
Comment (felipe300) [inline docs/src/pages/es/installation.md]: Change accepted

File: docs/src/pages/es/quick-start-md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+layout: ~/layouts/MainLayout.astro
+title: Comienzo rápido
+lang: es
+---
+
+```shell
+# prerrequisitos: revisa que Node.js está en la versión 12.20.0+, 14.13.1+, ó 16+
+node --version
+
+# Crea un nuevo proyecto en el directorio y entra al él
```
Comment (agustinmulet) [inline docs/src/pages/es/quick-start-md:11]: Please remove the l in "al"

File: docs/src/pages/es/quick-start-md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+npm install
+
+# comienza con el desarrollo
+npm run dev
+
+# cuando estés listo: crea tu sitio estático en la carpeta `dist/`
+npm run build
+```
+
+Para saber más sobre la instalación y uso de Astro por primera vez, por favor [lea nuestro guía de instalación.](installation)
```
Comment (agustinmulet) [inline docs/src/pages/es/quick-start-md:27]: Please change "nuestro" for "nuestra"

File: docs/src/pages/es/quick-start-md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Comienza con tu proyecto
+
+Dentro del directorio de tu proyecto, ingresa el siguiente comando en la terminal:
+
+```bash
+npm run dev
+```
+
+Astro comenzará a correr en el servidor de desarrollo en [http://localhost:3000](http://localhost:3000). Abre ésta ubicación en tu navegador, deberías ver la página de "¡Hola Mundo!" de Astro.
```
Comment (agustinmulet) [inline docs/src/pages/es/quick-start-md:39]: "Abre ésta ubicación en tu navegador", in this case, "esta" should not have accent

File: docs/src/pages/es/quick-start-md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+El servidor escuchará los cambios en vivo de los archivos en tu carpeta `src/`, así que, no necesitarás reiniciar la aplicación cuando hagas cambios durante el desarrollo.
+
+## Construye tu proyecto
+
+Para Construir tu proyecto, ingresa el siguiente comando en la terminal:
```
Comment (jgil-r) [inline docs/src/pages/es/quick-start-md:45]: Change `Construir` to `construir`

**Comment by jgil-r** [comment]
After those changes, looks good to me

File: docs/src/pages/es/quick-start-md (hunk 3 above)
Comment (felipe300) [reply docs/src/pages/es/quick-start-md]: Thanks, I'll fix it

File: docs/src/pages/es/quick-start-md (hunk 5 above)
Comment (felipe300) [reply docs/src/pages/es/quick-start-md]: Thanks, I'll fix it

File: docs/src/pages/es/quick-start-md (hunk 6 above)
Comment (felipe300) [reply docs/src/pages/es/quick-start-md]: Thanks, I'll fix it

**Comment by felipe300** [comment]
Done!

**Comment by matthewp** [comment]
Thanks everyone!

**Pull request merged**